src/CapabilityService.WebApi/Startup.cs

```diff
@@ -36,12 +41,28 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services
-                .AddMvc()
-                .SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Latest);
```

Comment:
Isn't it better to be explicit about version, eg. the one the solution is using (2.2)?